Content.Server/Atmos/Piping/Unary/EntitySystems/GasThermoMachineSystem.cs

```diff
@@ -78,25 +80,32 @@ private void OnGasThermoRefreshParts(EntityUid uid, GasThermoMachineComponent co
                 switch (part.PartType)
                 {
                     case MachinePart.MatterBin:
+                        nBins += 1;
                         matterBinRating += part.Rating;
                         break;
                     case MachinePart.Laser:
+                        nLasers += 1;
                         laserRating += part.Rating;
                         break;
                 }
             }
+            laserRating /= nLasers;
+            matterBinRating /= nBins;
```

Comment:
why dividing by the # of parts? just easier balancing?

Reply:
My reasoning is that it makes the yaml data-fields more intuitive, and it means you don't have to go back and update all the yaml definitions & data-field defaults if you decide a thermomachine should require 2 or 4 lasters/bins, instead of 3. It basically makes it not care about the number of parts, just the quality.

Reply:
sure